ex3-solution/src/components/layout/index.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+// order matters!
+import "./reset.css";
+import "./base.css";
+import "./layout.css";
+
+import Meta from "../../components/meta";
+import Logo from "../../components/logo";
+import Navigation, { navigationItems } from "../../components/navigation";
+
+import QuickLinks from "../../components/quick-links";
+import Teaser from "../../components/teaser";
+
+export const Container = ({ children }) => (
+  <div className="container">{children}</div>
+);
+
+export const Header = () => (
```

Comment:
so at this point i would split these up into separate ones and make them each have their own stylesheet, cause having one large layout file probably won't scale

Reply:
yep, added it as a note for their reference